#!/usr/bin/python3
def raise_exception_msg(message=""):
    """Function that raises a NameError with a custom message."""
    raise NameError(message)

